src/services/__tests__/QueueService.int.spec.ts

```diff
@@ -109,3 +127,28 @@ describe("QueueService tests", () => {
     });
   });
 });
+
+async function manuallyAddActiveMatch(activeMatch: PlayerInActiveMatch | Array<PlayerInActiveMatch>) {
```

Comment:
Is this function being used? If not, I think it'd be a good idea to remove it.

Reply:
Done